src/main/java/gov/usgs/earthquake/product/io/JsonProduct.java

```diff
@@ -0,0 +1,263 @@
+package gov.usgs.earthquake.product.io;
+
+import gov.usgs.earthquake.product.AbstractContent;
+import gov.usgs.earthquake.product.Content;
+import gov.usgs.earthquake.product.Product;
+import gov.usgs.earthquake.product.ProductId;
+import gov.usgs.earthquake.product.URLContent;
+import gov.usgs.util.CryptoUtils.Version;
+import gov.usgs.util.protocolhandlers.data.Handler;
+import gov.usgs.util.StreamUtils;
+import gov.usgs.util.XmlUtils;
+
+import java.math.BigDecimal;
+import java.net.URI;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonArrayBuilder;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+import javax.json.JsonValue;
+
+public class JsonProduct {
```

Comment:
Could this class/methods be turned into static access? Is there some benefit of `new JsonProduct().method()` everywhere? Could it just be `JsonProduct.method()` ?